README.md

```diff
@@ -691,8 +691,8 @@ If you're using the `X-SourceMap` header instead, you can just omit `sourceMap.u
   side effects permitting.
 
 - `comparisons` (default: `true`) -- apply certain optimizations to binary nodes,
-  e.g. `!(a <= b) → a > b` (only when `unsafe_comps`), attempts to negate binary
-  nodes, e.g. `a = !b && !c && !d && !e → a=!(b||c||d||e)` etc.
+  e.g. `!(a <= b) → a > b`, attempts to negate binary nodes, e.g.
+  `a = !b && !c && !d && !e → a=!(b||c||d||e)` etc.
```

Comment:
Both of these are invalid when either `a` or `b` are NaN, and I've intentionally used inequalities like the above in the past to deal with NaNs without explicitly coding for them. Specifically, both `a <= b` and `a > b` return `false` if either operand is a NaN.

Reply:
This of course does *not* involve side effects, just a simple differing result with primitives. You also get the same result with `undefined` as either parameter, which [per ToNumber is coerced to a NaN](https://tc39.es/ecma262/#sec-tonumber) in steps 4.b and 4.c of the [Abstract Relational Comparison `x < y`](https://tc39.es/ecma262/#sec-abstract-relational-comparison) abstract operation. (Note: they return `true` if less, `false` if greater, and `undefined` if equal in that algorithm.)